package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "my-app",
```

Comment:
nit: The app could be named something different, like `aiven-nextjs-netlify` etc.

Reply:
Fixed